app/routes/contact/components/ContactForm.js

```diff
@@ -57,14 +57,22 @@ const ContactForm = (props: Props) => {
     <Form onSubmit={props.handleSubmit(submit)}>
       <p>
         Dette skjemaet kan benyttes for å kontakte Abakus sine komiteer eller
-        Hovedstyret. Sender du til en spesifikk komité er det kun lederen av
+        Hovedstyret. Dersom du har opplevd avvik eller ubehagelige hendelser i
+        forbindelse med Abakus, ønsker vi gjerne at dette meldes fra om her.
+        Send oss gjerne også spørsmål, feedback eller ting du ønsker å fortelle
+        oss. Du kan også benytte dette skjemaet om du ønsker å avtale tid for
+        innsyn i HSP-avtalen.
```

Comment:
> Du kan også benytte dette skjemaet om du ønsker å avtale tid for innsyn i HSP-avtalen.

This felt like sort of out of place, following the more "serious" matters mentioned in the paragraph. Perhaps make it clear that it's just a random example, because I guess the point of it is to show how "anything" is welcome. Does that make sense?

Reply:
I now see that the example was there from previous - is it still even relevant? Maybe I'm overthinking.

Reply:
If I were to guess, I would think the example was originally added because people were confused about where they should contact Abakus in that exact case. (In my experience, when something is stated THAT clearly it's usually a reason for it)

But I kind of agree that it feels a bit out of place now that I added the part about "avvik". I could change the wording a bit so it's just one sentence about the serious stuff, and one about other things to use the contact form for. What do you think about: "Dersom du har opplevd avvik eller ubehagelige hendelser i forbindelse med Abakus, ønsker vi gjerne at dette meldes fra om her. Send oss gjerne også spørsmål, feedback eller ting du ønsker å fortelle oss, for eksempel om du ønsker å avtale tid for innsyn i HSP-avtalen"

Reply:
I like the new suggestion! Next to the other stuff, "HSP-avtalen" seemed like some holy thing within Abakus and felt weirdly special, although I understand it was just an absurd example.